client/logging/customLogger.go

```diff
@@ -0,0 +1,78 @@
+package logging
+
+import (
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+var log = logger.GetOrCreate("indexer/client/requests")
+
+type CustomLogger struct{}
+
+func (cl *CustomLogger) LogRoundTrip(
+	req *http.Request,
+	res *http.Response,
+	err error,
+	_ time.Time,
+	dur time.Duration,
+) error {
+	var (
+		nReq int64
+		nRes int64
+	)
+
+	if req != nil && req.Body != nil && req.Body != http.NoBody {
+		nReq, _ = io.Copy(ioutil.Discard, req.Body)
```

Comment:
rename `nReq` and `nRes` to something else

Reply:
done.